boot/freeldr/freeldr/include/ntldr/winldr.h

```diff
@@ -17,6 +17,31 @@
 // Some definitions
 
 #include <pshpack1.h>
+typedef struct /* ACPI Standard Header */
```

Comment:
I'm okay with duplicating the ACPI definitions you need it makes sense here, however let's place them in a hwacpi.h or something please.

Reply:
where should i put them?

Reply:
Probably in https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/drivers/acpi/acpi.h;hb=9b8c59d6b7214c61f1e9b76c7f7be9f4a8f8a61b

Reply:
It looks like we already have multiple definitions of these headers in multiple places. The best would be to unify this, but I don't want to make this more complicated than it needs to be now. I suggest you create a new header `freeldr/include/hwacpi.h` and we can improve on this later.

Reply:
> It looks like we already have multiple definitions of these headers in multiple places. The best would be to unify this, but I don't want to make this more complicated than it needs to be now. I suggest you create a new header `freeldr/include/hwacpi.h` and we can improve on this later.

Done!